packages/vite/src/node/env.ts

```diff
@@ -35,15 +35,27 @@ export function loadEnv(
     })
   )
 
-  // let environment variables use each other
-  const expandParsed = expand({
+  const expandOptions = {
     parsed: {
       ...(process.env as any),
       ...parsed
     },
     // prevent process.env mutation
     ignoreProcessEnv: true
-  }).parsed!
+  }
+
+  let expandParsed: NonNullable<ReturnType<typeof expand>['parsed']>
+  try {
+    // let environment variables use each other
+    expandParsed = expand(expandOptions).parsed!
+  } catch (e) {
+    if (e.message.includes('split')) {
```

Comment:
would be nice to add a comment here:
```suggestion
    // custom error handling until https://github.com/motdotla/dotenv-expand/issues/65 is fixed upstream
    // check for message "TypeError: Cannot read properties of undefined (reading 'split')"
    if (e.message.includes('split')) {
```